nrf-mpsl-sys/build.rs

```diff
@@ -27,22 +27,24 @@ struct Target {
     target: String,
     cpu: &'static str,
     float_abi: &'static str,
+    chip_family: &'static str,
     chip: &'static str,
 }
 
 impl Target {
     fn new(target: String) -> Self {
-        let (cpu, float_abi, chip) = match target.as_str() {
-            "thumbv7em-none-eabihf" => ("cortex-m4", "hard", "NRF52840_XXAA"),
-            "thumbv7em-none-eabi" => ("cortex-m4", "soft", "NRF52840_XXAA"),
-            "thumbv8m.main-none-eabi" => ("cortex-m33", "soft", "NRF5340_XXAA"),
+        let (cpu, float_abi, chip_family, chip) = match target.as_str() {
+            "thumbv7em-none-eabihf" => ("cortex-m4", "hard", "nrf52", "NRF52840_XXAA"),
```

Comment:
At least for mpsl, it makes more sense to use `NRF52_SERIES` and `NRF53_SERIES` defines.

Reply:
You can't just define the `_SERIES`, the header files require a specific chip to be selected. But the generated bindings are chip-independent (at least after #39)